tests/ops/op_math/test_exp.py

```diff
@@ -577,3 +581,123 @@ def test_draw_integration(self):
         qml.drawer.tape_text(tape)
 
         assert "0: ──Exp─┤  "
+
+
+class TestDifferentiation:
```

Comment:
Can we add tests which check that the gradient is compute correctly (in general and for parameter shift). I am not 100% sure, but if we can use parameter shift and also provide the input parameter in a different interface (eg. torch or jax). Might be worth adding a test for that as well.

Reply:
+1